extensions/2.0/Vendor/EXT_materials_specular_openpbr/README.md

```diff
@@ -0,0 +1,145 @@
+# EXT\_materials\_specular\_openpbr
```

Comment:
It would be safer to name the extension based on the actual interpretation difference rather than using a reference.

Reply:
I was expecting that issue to be raised but I'm not sure of a good name.
`EXT_materials_specular_edge_color`?
`EXT_materials_specular_f82`? - the F82 really only refers to the metallic model so probably isn't ideal.

Reply:
I've changed the name to `EXT_materials_specular_edge_color` for now.